BitwardenShared/UI/Vault/Vault/VaultGroup/VaultGroupView.swift

```diff
@@ -107,6 +107,11 @@ struct VaultGroupView: View {
                 groupView(with: items)
             }
         }
+        .overlay(alignment: .bottomTrailing) {
+            addItemFloatingActionButton {
+                store.send(.addItemPressed)
```

Comment:
⚠️ I think we sometimes don't show the add button in the toolbar, like if you're in the trash view. We should hide the FAB as well in that case. 
https://github.com/bitwarden/ios/blob/2990c57ea207b4c872097916f88b4ae92ad4cb3e/BitwardenShared/UI/Vault/Vault/VaultGroup/VaultGroupView.swift#L37-L38